frontend/src/components/Product/FarmerProductItem.vue

```diff
@@ -10,7 +10,7 @@
 
             <div v-if='!isEditing.price' style='margin-left: 5em;'>
                 ${{Number(product.price).toFixed(2)}}
-                <a @click='changeIsEditingPrice()' class='badge badge-success ml-3 mousePointer'> Quick Edit </a>
+                <a @click='changeIsEditingPrice()' class='badge badge-success ml-3 mousePointer' style='float:right;'> Quick Edit </a>
```

Comment:
Preferably change this to use flexbox, but at the very least, make this a class (psst: bootstrap has a built in class that does this already)